src/lib/common/sol-mainloop.c

```diff
@@ -76,6 +76,10 @@ extern void sol_comms_shutdown(void);
 extern int sol_update_init(void);
 extern void sol_update_shutdown(void);
 #endif
+#ifdef USE_UPDATE
```

Comment:
I'm trying to move out of ifdef, then please do the ifdef to declare as extern OR a static inline that is empty. In the call code, no ifdefs are required.